readme.md

```diff
@@ -3,3 +3,15 @@
 The official Laravel local development environment.
 
 Official documentation [is located here](http://laravel.com/docs/homestead).
+
+## ChangeLog
```

Comment:
I don't have access to the official documentation...  
How should I provide instructions?